src/main/java/com/sda/calc/ByFive.java

```diff
@@ -0,0 +1,30 @@
+package com.sda.calc;
+
+public class ByFive implements Series {
+    int startValue;
+    int acctualValue;
+
+
+    public int getNext(){
+        this.acctualValue+=5;
+        return this.acctualValue;
+    };
+    public int reset(){
+        this.acctualValue=this.startValue;
+        return this.acctualValue;
+    };
+    public void setStart(int x){
+        this.startValue=x;
+        this.acctualValue=x;
```

Comment:
Troszeczkę razi mnie niestandardowe formatowanie objawiające się brakiem spacji przed i po `=`. Polecam używać funkcję `reformat code` dostarczoną w Intellij.

Ta uwaga aplikuje się do reszty plików także. Polecam przyzwyczajać się do formatowania kodu, bo zwykle jest ono warunkiem koniecznym przy pracy w zespole. Po prostu jak nad kodem pracuje kilka osób, to zwykle dąży się by wyglądał on maksymalnie jednolicie.